doc/man/tracelog.3.txt

```diff
@@ -0,0 +1,154 @@
+tracelog(3)
+===========
+:object-type: macro
+
+
+NAME
+----
+tracelog - LTTng-UST man:printf(3)-like interface with a log level
+
+
+SYNOPSIS
+--------
+[verse]
+*#include <lttng/tracelog.h>*
+
+[verse]
+#define *tracelog*('level', 'fmt', ...)
+
+Link with `-llttng-ust`.
```

Comment:
`-ldl` is only required when defining `TRACEPOINT_DEFINE` or `TRACEPOINT_CREATE_PROBES` before including `<lttng/tracepoint.h>`, which is not the case with the include chain starting at `<lttng/tracelog.h>`.